snmp/datadog_checks/snmp/utils.py

```diff
@@ -287,7 +287,7 @@ def oid_dict(self, key, value):
             displayed = values[0]
         else:
             displayed = '{{{}}}'.format(', '.join(values))
-        return "'{}': {}".format(key, displayed)
+        return "'{}': {}".format(key, displayed.replace('\x00', ''))
```

Comment:
OK , tested and reproduced. Can you do it in the `oid_str_value` method as well? Thanks.

Reply:
Can you please explain what you found/what error is happening?

Reply:
I tested putting a null in the snmprec and running agent check against it. In debug mode it indeed failed with an error. I need to fix it in the other function to make it work as well.